inference-engine/src/transformations/src/transformations/op_conversions/convert_mvn1.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_mvn1.hpp"
+
+#include <ngraph/rt_info.hpp>
+
+#include <ngraph/opsets/opset2.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertMVN1, "ConvertMVN1", 0);
```

Comment:
Is this transformation triggered by anywhere?

Reply:
Nowhere right now